udata_front/theme/gouvfr/datagouv-components/src/components/DataserviceCard/DataserviceCard.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <article class="fr-my-2w fr-p-2w border border-default-grey fr-enlarge-link" :style="props.style">
+    <div class="absolute top-0 fr-grid-row fr-grid-row--middle fr-mt-n3v fr-ml-n1v" v-if="dataservice.private">
```

Comment:
Can you add an archived badge too ?

Reply:
It seems that archived doesn't exist on the dataservice type